test/byContract/IexecEscrow/IexecEscrowNative.ts

```diff
@@ -0,0 +1,280 @@
+// SPDX-FileCopyrightText: 2020-2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
+// SPDX-License-Identifier: Apache-2.0
+
+import { AddressZero } from '@ethersproject/constants';
+import { loadFixture, setStorageAt } from '@nomicfoundation/hardhat-network-helpers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { BigNumber } from 'ethers';
+import { ethers, expect } from 'hardhat';
+import CONFIG from '../../../config/config.json';
+import { loadHardhatFixtureDeployment } from '../../../scripts/hardhat-fixture-deployer';
+import { IexecInterfaceNative, IexecInterfaceNative__factory } from '../../../typechain';
+import { getIexecAccounts } from '../../../utils/poco-tools';
+
+const depositAmount = BigNumber.from(100);
+const depositNativeAmount = ethers.utils.parseUnits(depositAmount.toString(), 9);
```

Comment:
Since we have four occurrences, maybe it worth:
```suggestion
const depositNativeAmount = toNativeAmount(depositAmount);
```
or
```
const .. = 9;
```

Reply:
Done with toNativeAmount function